### PR TITLE
Parameterize mosaic

### LINKED ISSUE #5: Parameterize mosaic script
The status queue name (SQS) should be a command line argument to this script.

### PR DESCRIPTION
**Pull request by echeipesh**
```bash
mosaic 0.1
Usage: scopt [options] <chunker result>

  <chunker result>
        URL or local path of chunker json output
  --status-queue <value>
        SQS URL
```

This still reads from `application.conf` although I think it's up for debate if that has any further value.

Fixes #5

### CODE REVIEW

**Comment by kdeloach** [comment]
+1

**Comment by kdeloach** [comment]
We could also add "target" as a command line argument in the future. As of now, it exists as an argument to chunk.py and it does nothing except populate a value in the chunker result JSON file.

File: mosaic/src/main/scala/com/azavea/rasterfoundry/Main.scala [hunk 1]
```diff
@@ -22,30 +22,26 @@ object Main {
...
   def main(args: Array[String]): Unit = {
-    val publishNotifications =
-      args.length != 2
-
-    if(args.length < 1) {
-      sys.error("Argument error: must give the URI to the step1 result JSON.")
-    }
-
+    val config = ConfigParser.parse(args, Config()).getOrElse(sys.exit())
+    val status = new Status(config.statusQueue, config.publishNotifications)
```
Comment (kdeloach) [inline mosaic/src/main/scala/com/azavea/rasterfoundry/Main.scala]: Can we assume `publishNotifications` is true if `statusQueue` is non-empty? If so, we can remove the `publishNotifications` argument.

Comment (echeipesh) [inline mosaic/src/main/scala/com/azavea/rasterfoundry/Main.scala]: Actually not, there is always a default argument for SQS queue that will be picked from `application.conf`. I would suggest getting rid of those configurations and lean only on command line args. In which case what you say is true. @hectcastro do you have a preference on how things should be?

Comment (kdeloach) [inline mosaic/src/main/scala/com/azavea/rasterfoundry/Main.scala]: In any case, we will always want to publish notifications...

Comment (echeipesh) [inline mosaic/src/main/scala/com/azavea/rasterfoundry/Main.scala]: Removed that option to keep consistency with chunk.py. Also added some logging for SQS URL being used and Target read from the input file.

**Pull request merged**